docs/public/guidelines/languages/go.md

```diff
@@ -0,0 +1,222 @@
+# Go guidelines
+
+-----
+
+## Imports
+
+The imports defined in the `imports ( ... )` block of each Go file should be separated into the following sections, in order.
+
+1. Standard library packages (e.g. `fmt`, `net/http`)
+1. External packages (e.g. `github.com/stretchr/testify/assert`, `github.com/DataDog/datadog-agent/pkg/util/log`)
+1. [Internal](#public-apis) packages (e.g. `github.com/DataDog/datadog-agent/<parent>/internal`)
+
+This is not verified by our [static analysis](../../how-to/test/static-analysis.md) as doing so is very inefficient. Instead, configure your editor to keep imports properly sorted.
```

Comment:
I wouldn't mention the part that is inefficient, as when having it set in the editor and saving the file, it does not take much time. 

What about something like: 

> This is not verified by our [static analysis](../../how-to/test/static-analysis.md) during CI. Instead, we suggest that you configure your editor to keep imports properly sorted.


Also, we should mention configuring the editor to format on save 😄